process-engine/bpmn-studio
# ✨Refactor ProcessDefList

## What did you change?

This branch refactors the "process-def-list" component drastically.
It also adds a first draft of a real start page kinda like VSCode shows when no file was opened.

## How can others test the changes?

Start BPMN-Studio in Webversion. Click on "Plan" to see the refactored process-def-list.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
